deploy/packer/app/scripts/deploy.sh

```diff
@@ -33,3 +33,7 @@ sudo npm install -g babel@4.7.16
 sudo sh -c 'echo "v4l2loopback" >> /etc/modules'
 sudo sh -c 'echo "snd-aloop" >> /etc/modules'
 sudo depmod
+
+# install busybox
+sudo curl -o /worker/bin-utils/busybox http://www.busybox.net/downloads/binaries/latest/busybox-x86_64
```

Comment:
I suggested that we move into the `postinstall` hook for `npm install`... That way it'll always be there when we've done npm install... And it'll in in the ci environment...